runhouse/rns/rns_client.py

```diff
@@ -632,18 +642,16 @@ def contents(self, name_or_path, full_paths):
             full_paths=full_paths
         )
 
-    async def send_status(self, status: ResourceStatusData, cluster_rns_address: str):
+    async def send_status(
```

Comment:
why do we need to pass around `cluster_uri` and api_server_url` here, don't we have it in the `cluster_config`?

Reply:
@jlewitt1 we have the cluster name, and we need to modify it to be in the URI format. Since we need the URI both for cluster status check and the logs surfacing, I thought it made sense to modify it once and send it to both methods as arguments. wdyt?